src/blocks/BoardListBlock.tsx

```diff
@@ -1,38 +1,211 @@
+import classnames from "classnames";
 import { extractCompounds } from "utils/compound-utils";
+import { faArrowAltCircleRight } from "@fortawesome/free-regular-svg-icons";
 
 const Empty: React.FC<unknown> = ({ children }) => <p>{children}</p>;
 
 const Item = ({
   slug,
+  avatar,
+  color,
   description,
+  link,
   _selected,
   _onSelected,
   options,
+  updates,
+  muted,
+  outdated,
+  _onPinned,
+  _onMuted,
 }: ItemProps) => (
   <>
-    <button className="slug-container" onClick={() => _onSelected?.(slug)}>
-      !{slug}
-    </button>
-    {options.length !== 0 && (
-      <DropdownMenu options={options} zIndex={200} label="board list options">
-        <div className="board-list-options">
-          <FontAwesomeIcon icon={faEllipsisV} />
+    <div className="item-summary">
+      <ActionLink link={link}>
+        <BoardIcon
+          avatar={avatar}
+          color={color}
+          updates={updates}
+          muted={muted}
+          outdated={outdated}
+          large
+        />
+      </ActionLink>
+      <button
+        onClick={() => _onSelected?.(slug)}
+        className={classnames("slug-container", {
+          "has-updates": !!updates,
+          muted: !!muted,
+          outdated: !!outdated,
+          "with-options": !!options.length,
+        })}
+      >
+        <span className="slug">!{slug}</span>
+      </button>
+      {options.length !== 0 && (
+        <DropdownMenu
+          options={options}
+          zIndex={200}
+          label="board list options"
+          buttonClassName="insanity"
+          style={DropdownStyle.DARK}
+        >
+          <FontAwesomeIcon
+            icon={faEllipsisV}
+            size="lg"
+            color={DefaultTheme.MENU_ITEM_ICON_COLOR}
+          />
+        </DropdownMenu>
+      )}
+    </div>
+    {_selected && (
+      <div className="item-details">
+        <BoardPreview
+          slug={slug}
+          avatar={avatar}
+          description={description}
+          color={color}
+          muted={muted}
+          displayStyle={DisplayStyle.MINI}
+        />
+        <div className="board-info">
+          <HighlightedText highlightColor={color}>
+            <span className="big-slug">!{slug}</span>
+          </HighlightedText>
+
+          {description}
+          <div className="board-info-footer">
+            <div className="toggle-buttons">
+              <CircleButton
+                icon={{ icon: faThumbTack }}
+                defaultBorderColor={color}
+                link={{ onClick: _onPinned }}
+              />
+              <CircleButton
+                icon={{ icon: muted ? faVolumeMute : faVolumeHigh }}
+                defaultBorderColor={color}
+                link={{ onClick: _onMuted }}
+              />
+            </div>
+              <ActionLink link={link}>
+                Go to Board{" "}
+                <FontAwesomeIcon
+                  icon={faArrowAltCircleRight}
+                  size="lg"
```

Comment:
I don't generally use these props and style the svg directly using `:global(svg)`. Mostly useful since our own Icon does not expose size.